core/database_arango.py

```diff
@@ -1185,13 +1290,19 @@ def filter(
         if sorts:
             aql_sort = f"SORT {', '.join(sorts)}"
 
+        acl_filter = ""
+        if acl_query:
+            acl_filter = "FILTER acl"
+
         aql_string = f"""
             FOR o IN @@collection
```

Comment:
We need to add WITH statement here for cluster deployments

Reply:
Good catch, will add that to the query.